schemas/put-artifact-response.yml

```diff
@@ -0,0 +1,13 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Put Artifact Response"
+description: complete a blob artifact upload
+type:               object
+properties:
+  outcome:
+    type: string
+    enum:
+      - success
+      - failure
```

Comment:
sometimes clients have bad debugging options and don't expose the status code in an easy way.  the idea here is that we're transferring an extra 20? bytes which could make debugging eaiser.